polls/polls.js

```diff
@@ -0,0 +1,137 @@
+import { checkUser, getPolls, getUserId, logOutUser, savePoll } from '../utils/fetch-utils.js';
+import { displayCurrentPoll, renderPastPolls } from '../utils/render-utils.js';
+
+const newPoll = document.getElementById('new-poll');
+
+const option1Votes = document.getElementById('first-count');
+const option2Votes = document.getElementById('second-count');
+
+const upVote1 = document.getElementById('first-option-btn-plus');
+const downVote1 = document.getElementById('first-option-btn-minus');
+
+const upVote2 = document.getElementById('second-option-btn-plus');
+const downVote2 = document.getElementById('second-option-btn-minus');
+
+const closePoll = document.getElementById('submit-poll');
+
+const logOut = document.getElementById('logout-btn');
+
+let questionState = '';
+let option1Title = '';
+let option1VoteCount = 0;
+let option2Title = '';
+let option2VoteCount = 0;
+
+window.addEventListener('load', async() => {
+    checkUser();
+    const user = await getUserId();
+    const closedPolls = await getPolls(user);
+    renderPastPolls(closedPolls);
+});
+
+logOut.addEventListener('click', async() => {
+    await logOutUser();
+});
+
+newPoll.addEventListener('submit', (e) => {
+    e.preventDefault();
+
+    const poll = new FormData(newPoll);
+    const question = poll.get('question');
+    questionState = question;
+
+    const option1 = poll.get('option-one');
+    option1Title = option1;
+
+    const option2 = poll.get('option-two');
+    option2Title = option2;
+
+    const currentPoll = {
+        questionState,
+        option1Title,
```

Comment:
it looks like this is where you need to reset the votes to 0, otherwise it loads a "new poll" with whatever votes belonged to the previous poll

Reply:
I indeed overlooked this. I will pay closer attention in the future.